src/WebService/Http/CurlRequest.php

```diff
@@ -53,11 +68,16 @@ public function get()
      */
     private function createCurl()
     {
-        $curl = curl_init($this->url);
+        curl_reset($this->ch);
```

Comment:
If passing it in is part of the API, I wonder if we should be resetting it?

Reply:
Good point. I don't know why we would need it. Maybe we should move all of the options not related to the particular request to where we create the handle too.

Reply:
I started moving all of the options to where we create the handle, but realized that would cause ripple effects for the tests in the API packages. I don't think the change is worth it without a more compelling reason.

I also realized that we are not passing the options straight through so the `empty` wouldn't do anything anyway. I think I'll remove that functionality.